src/com/aca/Armine Arzumanyan/HomeWork_1/src/Gadget.java

```diff
@@ -0,0 +1,57 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Gadget implements Item {
+    public String name;
+    public int code;
+    public int price;
+    public int recDuration;
+    public boolean mp4;
+    public boolean mp3;
+    public boolean hd;
+    public boolean vga;
+    static List<Gadget> allItems = new ArrayList<>();
```

Comment:
Gadget is a simple product like a camera pen or smth else and is it ok to put into it _allItems_ and _soldItems_ properties. The mentioned properties are related to the manager mostly, some **manager** should care about sold and existing items and not simple item